### DISCUSSION TITLE
Does AGridder MidiFX in Logic Pro receive midi back from AG Server hosted plugins that output midi?

### DISCUSSION THREAD

**Discussion by eakwarren** [apohl79/audiogridder#757] [Help]
Hi AG Team,

Does AGriddder 1.2.0b5 loaded as a **MidiFX** in Logic Pro 10.6.3 receive midi **_back_** from AG Server 1.2.0b5 hosted plugins that output midi? I'm running AG Server on a Win10 box with the free CC Stepper plugin (https://www.codefn42.com/ccstepper/index.html) or BlueArp (http://graywolf2004.net/wp/). I inserted a midi monitor after the Agridder midiFX in Logic and it isn't showing activity. Am I doing something incorrectly?

Thanks!

**Comment by trevormeier** [comment]
I'd also like to use AG as a MidiFX if it's possible to implement.
